repo-scripts/size-analysis/index.ts

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
I'm sure there's a reason, but just asking for my own clarification, what is the purpose of this? If I read correctly, this isn't code used for the actual analysis script, it's something that builds as part of a full repo build, and functions as a sort of test to make sure the package builds without breaking? And there's also a separate karma test file `index.test.ts` that runs with `yarn test`. Do these cover different things?